libxdfConfig.cmake.in

```diff
@@ -0,0 +1,9 @@
+@PACKAGE_INIT@
+
+include(CMakeFindDependencyMacro)
+
+if(NOT TARGET XDF::xdf)
+    include("${CMAKE_CURRENT_LIST_DIR}/libxdfTargets.cmake")
+endif()
+
+find_dependency(pugixml)
```

Comment:
This line should be moved up. If `pugixml` isn't found, `include("${CMAKE_CURRENT_LIST_DIR}/libxdfTargets.cmake")` shall not be called.

Reply:
Hi! @dg0yt Thanks for your review. That sounds reasonable to me.